scrapers/GammaEntertainment.yml

```diff
@@ -117,7 +121,6 @@ movieByURL:
       - darkx.com/en/movie/
       - eroticax.com/en/movie/
       - evilangel.com/en/movie/
-      - famedigital.com/en/
```

Comment:
this was added last week <https://github.com/stashapp/CommunityScrapers/pull/821>
why is it removed?